hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataWriter.java

```diff
@@ -40,4 +41,9 @@
   void update(HoodieRestoreMetadata restoreMetadata, String instantTime);
 
   void update(HoodieRollbackMetadata rollbackMetadata, String instantTime);
+
+  /**
+   * Return the timestamp of the latest instant synced to the metadata table.
+   */
+  Option<String> getSyncedInstantTime();
```

Comment:
`getLatestSyncedInstantTime` is better?

Reply:
Sounds better. Renamed to getLatestSyncedInstantTime.